src/genie/libs/parser/iosxe/tests/ShowIpBgpAllNeighbors/cli/equal/golden_output2_expected.py

```diff
@@ -0,0 +1,562 @@
+                        "iss": 3531560020,
+                        "krtt": 0,
+                        "lock_slow_path": 0,
+                        "max_rtt": 1000,
+                        "maximum_output_segment_queue_size": 50,
+                        "maxrcvwnd": 16384,
+                        "min_rtt": 3,
+                        "min_time_between_advertisement_runs": 0,
+                        "minimum_incoming_ttl": 0,
+                        "option_flags": "VRF "
+                        "id "
+                        "set, "
+                        "nagle, "
+                        "path "
+                        "mtu "
+                        "capable",
+                        "outgoing_ttl": 1,
+                        "packet_fast_path": 0,
+                        "packet_fast_processed": 0,
+                        "packet_slow_path": 0,
+                        "rcv_scale": 0,
+                        "rcvnxt": 1970767239,
+                        "rcvwnd": 16346,
+                        "receive_idletime": 19547,
+                        "rib_route_ip": "10.10.11.1",
+                        "rtto": 1009,
+                        "rtv": 10,
+                        "sent_idletime": 19347,
+                        "snd_scale": 0,
+                        "sndnxt": 3531561124,
+                        "snduna": 3531561124,
+                        "sndwnd": 32768,
+                        "srtt": 999,
+                        "sso": False,
+                        "status_flags": "active " "open",
+                        "tcp_path_mtu_discovery": "enabled",
+                        "tcp_semaphore": "0x7F3C800B6268",
+                        "tcp_semaphore_status": "FREE",
+                        "transport": {
+                            "foreign_host": "10.10.11.1",
+                            "foreign_port": "179",
+                            "local_host": "10.10.11.2",
+                            "local_port": "39949",
+                            "mss": 1024,
+                        },
+                        "unread_input_bytes": 0,
+                        "uptime": 1281819,
+                    },
+                    "bgp_version": 4,
+                    "link": "external",
+                    "local_as": 4200000101,
+                    "no_prepend": True,
+                    "remote_as": 4200000001,
+                    "replace_as": True,
+                    "router_id": "10.0.1.1",
+                    "session_state": "Established",
+                    "shutdown": False,
+                },
+                "10.10.21.6": {
+                    "address_family": {
+                        "ipv4 multicast": {},
+                        "l2vpn evpn": {},
+                        "mvpnv4 unicast": {},
+                        "mvpnv6 unicast": {},
+                        "vpnv4 flowspec": {},
+                        "vpnv4 multicast": {},
+                        "vpnv4 unicast": {
```

Comment:
Values are missing or why there is an empty dict?

Reply:
Same as first comment - this is expected from output of neighbor does not negotiage corresponding BGP AF